src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/entity/CalculationOutcome.kt

```diff
@@ -0,0 +1,26 @@
+package uk.gov.justice.digital.hmpps.calculatereleasedatesapi.entity
+
+import com.vladmihalcea.hibernate.type.json.JsonBinaryType
+import org.hibernate.annotations.TypeDef
+import org.hibernate.annotations.TypeDefs
+import java.time.LocalDateTime
+import java.util.UUID
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.Table
+
+@Entity
+@Table
+@TypeDefs(
+  TypeDef(name = "jsonb", typeClass = JsonBinaryType::class)
+)
```

Comment:
As there is no JSONB types in this entity it probably doesn't need the @Typedefs?

Reply:
good spot - copy/paste error